en/style/colors.md

```diff
@@ -1,93 +1,64 @@
 ---
 
 # Colors
 
-Use Colors from the Styling library file to set up your theme's colors - `primary`, `secondary`, `success`, `warn`, `error`, `info`. The way Colors are set up in the Styling library is identical to the [Ignite UI for Angular Themes](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes.html).
+On the `🎨 Colors` page of the **Indigo.Design System** you will find a collection of `primary`, `secondary`, `success`, `warn`, `error`, `info`, `grays`, and `series` colors to let you brand the library all at once. The way Colors are set up is identical to the [Ignite UI for Angular Themes](https://www.infragistics.com/products/ignite-ui-angular/angular/components/themes.html).
 
-<img class="responsive-img" src="../images/colors_palette.png" srcset="../images/colors_palette@2x.png 2x" />
+<img class="responsive-img" src="../images/colors_palette.png" />
 
-## Palette Generation
-
-To change the primary color, navigate to the `Colors` page in Indigo-Styling and while holding the `command` key select all the primary color variants. Then, find the Fills section in the right panel and click on the bottom one, which bears the main color component in order to change its value to another color. This will automatically update all primary variants to assure consistent look with Ignite UI for Angular.
-
-<img class="responsive-img" src="../images/colors-fills.png"/>
-
-> [!Warning]
-> The two fills above are used for color variant generation and their values must always be kept as they are. Failing to comply with this guidance will result in inconsistencies and unpredictable code generation results.
-
-Upon changing the bottom fill, you will see the whole primary palette instantly update. The same palette generation is available for the secondary color.
-
-<img class="responsive-img" src="../images/colors_generation.png" srcset="../images/colors_generation@2x.png 2x" />
-
-With the introduction of Library Styles in Sketch and our support for Shared Color Styles there is one more step that you need to do i.e. to update all respective color styles that have changed with the selection of a new primary/secondary color. In order to do that upon selecting an individual color element, you should be able to observe the following inconsistency.
-
-<img class="responsive-img" src="../images/colors_styles_update.png"/>
+The **Indigo.Design System** version 3 introduces the concept of series palettes with two collections of ten colors each optimized for various data visualization scenarios. Both palettes ensure good contrast between adjacent series colors so that the palette can be used e.g. to colorize ranges of a gauge. One of the palettes is further optimized to be 100% inclusive and facilitate easy comprehension by people with color vision defficiencies. 
```

Comment:
The **Indigo.Design System** version 3 - is that official version of Indigo?

Reply:
YES